R/pmcmc.R

```diff
@@ -696,3 +699,165 @@ plot.pmcmc <- function(x, ...) {
   
   
 }
+
+##' @export
+##' @importFrom viridis cividis
+##' @importFrom graphics hist par plot.new text lines legend
+##' 
+plot.pmcmc_list <- function(x, burn_in = 1, ...) {
+  
+  summ <- summary(x, burn_in = burn_in)
```

Comment:
Do you think burn in is better as a number of samples or as a proportion of the total? I did it as the proportion in ncov outputs (unless that was changed over the weekend), forgetting that you'd been using number here. It would probably be best for us to choose one way and stick with it – should it be number of samples?

Reply:
Hmm there are pros and cons to both... Shall we go with a fixed number of samples, because the part of the chain we want to discard does not vary with chain length? This will annoyingly necessitate checks on the respective lengths but I'll add those as I go.

Reply:
Sounds good – I think having the function you added above helps too, as everything should go through there now